web/client/components/background/BackgroundDialog.jsx

```diff
@@ -210,6 +214,26 @@ export default class BackgroundDialog extends React.Component {
                 </Button>
             </React.Fragment>);
         }
+        if (this.props.layer.type === "wmts") {
+            return (
+                <React.Fragment>
+                    <FormGroup controlId="formControlsSelect">
+                        <ControlLabel><Message msgId="backgroundDialog.editAttribution" /></ControlLabel>
+                        <Editor
+                            modules={{
+                                toolbar: [
+                                    ['bold', 'italic', 'underline'],
+                                    ['clean'], ['link']
+                                ]
+                            }}
+                            placeholder="backgroundDialog.editAttributionPlaceholder"
+                            value={ this.state.attribution || '' }
+                            onChange={text => this.setState({ attribution: text || '' })}
+                        />
```

Comment:
If I rememeber correctly we are trying to remove the quill library.
Could we make a quick test with the other editor library we are unsing in geostory?

`import { Editor } from 'react-draft-wysiwyg';`

This change is not mandatory due the fact we are still using quill in other place I was just wondering if we still want to give priority to the react-draft-wysiwyg' editor cc @offtherailz